modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePaymentMethod.testcase

```diff
@@ -0,0 +1,388 @@
+		}
+
+		task ("And give the buyer places an order using the first offline payment method") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "1");
+
+			CommerceNavigator.gotoMiniCart();
+
+			Button.click(button = "View Details");
+
+			Button.click(button = "Checkout");
+
+			CommerceCheckout.initializersCheckout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				paymentMethod = "Test Offline Payment Method 1",
+				productName = "U-Joint",
+				productQuantity = "1",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
+
+		task ("When the admin goes to the Orders page to view the pending order") {
+			User.logoutAndLoginPG();
+
+			CommerceOrders.openOrdersAdmin();
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "Pending");
+		}
+
+		task ("Then the selected offline payment method is present") {
+			CommerceOrders.viewOrdersEntry(
+				addressAddress = "Test Address",
+				addressCity = "Test City",
+				addressZipCode = "Test Zip Code",
+				paymentMethod = "Test Offline Payment Method",
+				productListPrice = "24.00",
+				productName = "U-Joint",
+				productQuantity = "1",
+				productSku = "MIN55861",
+				productTotalPrice = "24.00");
+		}
+
+		task ("When the admin changes the payment method to another offline payment method") {
+			CommerceOrders.editPaymentMethod(paymentMethod = "Test Offline Payment Method 2");
+		}
+
+		task ("Then the payment method should be updated") {
+			AssertTextEquals.assertPartialText(
+				locator1 = "CommerceOrders#ORDER_DETAILS_PAYMENT_METHOD_NAME",
+				value1 = "Test Offline Payment Method 2");
+		}
+	}
+
+	@description = "Verify an activated offline payment method can be selected for an order when multiple payment methods are active."
+	@priority = "3"
+	test CreateOrderWithMultipleActivePaymentMethods {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site with a buyer user") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+
+			User.openUsersAdmin();
+
+			CommerceLogin.editPasswordCP(
+				newPassword = "test",
+				requirePasswordReset = "false",
+				userScreenName = "buyeruser");
```

Comment:
```suggestion

```
This editPasswordCP will not work since the password will be set as "test" automatically when calling `CommerceEntry.addAccountEntryUser()` macro.